docs/administration/upgrade/v6.0_changes.adoc

```diff
@@ -566,97 +480,162 @@ spec:
+      authentication:
+        token:
+          from: serviceAccount
+    tls:
+      ca:
+        key: service-ca.crt
+        configMapName: openshift-service-ca.crt
+  pipelines:
+  - name: my-pipeline
+    outputRefs:
+    - default-lokistack
+    inputRefs:
+    - application
+    - infrastructure
+    - audit
 ----
-.v5 ClusterLogForwarder
+====
+
+
+=== Forwarding to the External Elasticsearch
+====
+.Complete spec including url, version and custom index
 [source, yaml]
 ----
-apiVersion: logging.openshift.io/v1
+apiVersion: observability.openshift.io/v1
 kind: ClusterLogForwarder
 metadata:
-  name: instance
-  namespace: openshift-logging
+  name: my-forwarder
 spec:
+  serviceAccount:
+    name: my-user
   outputs:
-    - name: default
-      type: elasticsearch
-      elasticsearch:
-        enableStructuredContainerLogs: true
-        structuredTypeKey: kubernetes.namespace_name
-        structuredTypeName: nologformat
+  - name: es-external
+    type: elasticsearch
+    elasticsearch:
+      url: https://external-es-service:9200
+      version: 8
+      index: '{.log_type||"nologformat"}-write'
+    tls:
+      ca:
+        key: bundle.crt
+        secretName: my-tls-secret
+      certificate:
+        key: tls.crt
+        secretName: my-tls-secret
+      key:
+        key: tls.key
+        secretName: my-tls-secret
   pipelines:
-    - inputRefs:
-        - application
-      name: to-default
-      parse: json
-      outputRefs:
-        - default
-----
-.v6 ClusterLogForwarder
+  - name: my-pipeline
+    inputRefs:
+    - application
+    - infrastructure
+    filterRefs:
+    - my-parse
+    outputRefs:
+    - es-managed
+----
+`index` can be a combination of dynamic and static values. Dynamic values are enclosed in curly brackets `{}`
+and MUST end with a "quoted" static fallback value separated with `||`.
+
+More details use: `oc explain clf.spec.outputs.elasticsearch.index`
+
+NOTE: In this example, application logs are written to the 'application-write' and 'infrastructure-write' index.
+Previous versions without the `index` spec, would have instead written to 'app-write' and 'infra-write'.
+====
+
+
+=== Forwarding to the Red Hat Managed Elasticsearch
+====
+.Complete spec including url, version and static index values (required)
 [source, yaml]
 ----
 apiVersion: observability.openshift.io/v1
 kind: ClusterLogForwarder
 metadata:
-  name: instance
-  namespace: openshift-logging
+  name: my-forwarder
 spec:
   serviceAccount:
     name: logcollector
   outputs:
-    - name: es-managed
-      type: elasticsearch
-      elasticsearch:
-        url: https://elasticsearch:9200
-        version: 6
-        index: '{.kubernetes.namespace_name||"nologformat"}-write'
-      tls:
-        ca:
-          key: ca-bundle.crt
-          secretName: collector
-        certificate:
-          key: tls.crt
-          secretName: collector
-        key:
-          key: tls.key
-          secretName: collector
+  - name: es-managed-app
```

Comment:
I just thought about this, we could be clever and allow a single output def that relies upon an openshift.label to craft the index.  We create 2 pipelines to the same output and use the openshiftLabel filter on each pipeline to set "app" or "infra" and then use that for the index name

Reply:
That is clever :)

Reply:
Perfect, and I was also thinking, if they have created custom indexes using the 'structuredType' options in 5.9, we may want to show how to achieve that as well.

Reply:
Updated, please review.